### ISSUE TITLE
email validation trouble

### ISSUE THREAD

**Issue by amelon** [poppinss/indicative#17]
first of all, nice project, it makes me happy with json validation :)

My point here is I want to validate email format only if my field has data.

``` javascript
var indic = new(require('../lib/indic_validator'));

var schema = {
  email: 'email'
};

var data = {
  email: null
};

indic.validateAll(schema, data)
.then(function() { console.log('ok') })
.catch(function() { console.log('bad email format') });

// log "bad email format"
// I expect "ok"
```

The problem is in this case, I get an error even if there is no data in email field.

I just want to get an error if there is data in email.

Is it possible to handle this ?

**Comment by thetutlage** [comment]
yes ur use case is valid and ideally email validation should be ignoring it.

Let me check

**Comment by thetutlage** [comment]
Fixed as of `v1.2.6`
